.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Based on the activity of this project, its seems more appropriate to check it less often.

```suggestion
      interval: "weekly"
```